gulpfile.js

```diff
@@ -250,7 +250,9 @@ gulp.task('js:libs', ['appconfig'], function() {
             "./src/lib/angular-moment/angular-moment.js",
             "./src/lib/ngImgCrop/compile/unminified/ng-img-crop.js",
             "./src/lib/qrcode/lib/qrcode.js",
-            "./src/lib/angular-qr/src/angular-qr.js"
+            "./src/lib/angular-qr/src/angular-qr.js",
+
+            "./src/lib/fingerprintjs2/dist/fingerprint2.min.js"
```

Comment:
whenever possible don't use minified libs since we minify it ourselves at a later stage (in this case use `./src/lib/fingerprintjs2/fingerprint2.js`)